CHARTER.md

```diff
@@ -309,6 +310,47 @@ o. Any issues that cannot be resolved by the TOB shall be referred to The Linux
    Foundation Executive Director for mediation, and, if necessary, for
    resolution by The Linux Foundation Board of Directors.
 
+p. OCI Working Groups.
+
+   - i. The OCI may establish one or more working groups (each, an “OCI Working
+     Group”) to experiment, validate, and define new OCI Specifications or
+     major revisions to existing OCI Specifications. Each OCI Working Group
+     shall have a defined scope of the OCI Working Group's intended activities
+     and goals. The end result of the OCI Working Group's work product shall be
+     either a new OCI Specification, an update to an existing OCI
+     Specification, or a new OCI Project.
+
+   - ii. Any participant in the OCI technical community may propose a working
+     group to the TOB for consideration. The TOB shall maintain a public list
+     of the information required to be included in a working group proposal.
+
+   - iii. Approval of a new OCI Working Group requires a two-thirds vote of the
+     TOB. The TOB will maintain a public list of all approved OCI Working
+     Groups.
+
+   - iv. Once an OCI Working Group determines that it has achieved its
+     objectives, the OCI Working Group leaders shall present to the TOB the
```

Comment:
Hmmm... I was thinking that this read more as saying that this would happen at the end of the drafting process (when the WG "determines that it has achieved its objectives"). At least that was my intent! Do you have a suggestion for a clearer way to phrase this?

Reply:
Maybe add a step before it to prompt to define objectives, channels of communication and (if appropriate) timelines? E.g., some WG might just be working on a text document, and others might want a toy implementation. Then leading into this section would follow nicely.